<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZoomableChartWidget</class>
 <widget class="QWidget" name="ZoomableChartWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayoutToolbar">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonFitInView">
       <property name="toolTip">
        <string>Adjust zoom to fit all</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="res/zoomable_chart_widget_resources.qrc">
         <normaloff>:/icons/zoom_to_fit_all.png</normaloff>:/icons/zoom_to_fit_all.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxZoomMode">
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="ZoomableChartView" name="chartView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomableChartView</class>
   <extends>QGraphicsView</extends>
   <header>ZoomableChartView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res/zoomable_chart_widget_resources.qrc"/>
 </resources>
 <connections/>
</ui>
